sv_pipeline/genome/utils/mapping_gene_ids.py

```diff
@@ -0,0 +1,77 @@
+import gzip
+import logging
+import os
+import pickle
+from tqdm import tqdm
+
+from sv_pipeline.genome.utils.download_utils import download_file
+
+GENOME_VERSION_GRCh37 = "37"
+GENOME_VERSION_GRCh38 = "38"
+
+logger = logging.getLogger(__name__)
+
+GENCODE_GTF_URL = "http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_{gencode_release}/gencode.v{gencode_release}.annotation.gtf.gz"
+
+# expected GTF file header
+GENCODE_FILE_HEADER = [
+    'chrom', 'source', 'feature_type', 'start', 'end', 'score', 'strand', 'phase', 'info'
+]
+
+
+def load_gtf_data(gene_id_mapping, gencode_gtf_path):
```

Comment:
the way you've split up the functionality is a bit weird. The things you do in the loading code here are 
1) get the file path for the gencode data and download it if its missing
2) get the file path for the pickle file and check if you have pickled data
3) actually parse the gencode data if you didn't have it pickled

For one thing, the order of 1 and 2 should be swapped a bit - why download the file from gencode if you already have the parsed data pickled and aren't going to use it? You've also put 1 in one function and 2 and 3 in the other. I think it makes much more sense to put 1 and 2 together and have 3 be the separate helper

Reply:
Sounds good.